docs/python-examples/fixtures/test_fixtures.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+def alter_pvc(pvc):
+    """
+    Dummy helper function to alter pvc
+
+    Args:
+        pvc (PVC): PVC object reference
+    """
+    logger.info(f"Doing something with pvc: {pvc}")
+
+
+@pytest.fixture(scope='class')
+def three_pvcs(pvcs_factory, storage_class):
+    """
+    This is fixture related just to one specific test in this module so no
+    need to have it defined in conftest
+    """
+    return pvcs_factory(3, storage_class)
+
+
+class TestCreatingPVCsFromTest:
+    # This needs to be discussed! How to solve the issue of sharing resources
```

Comment:
Not sure I understand the question, what is the issue with sharing resources between tests when the fixture is class scope?

Reply:
This comment is here to discuss the approach we are discussing in next comment on line 47.